.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Release monorepo
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseParams:
+        description: 'Pass parameters to lerna publish (ex: major, minor, patch)'
+        required: false
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
@oltodo Is this the correct name of the secret ?

Reply:
It is.